File: PostApp/src/app/Models/comment.model.ts

export class CommentModel {
    id: string;
    authorId: string | null;
    postId: string;
    content: string;
    creationDate: string;


    constructor() {
        this.id = "";
        this.authorId = "";
        this.postId = "";
        this.content = "";
        this.creationDate = "";
    }
}

File: TESTAPI1/Models/Comment/CommentModel.cs
namespace TEST_API1.Models.Comment
{
    public class CommentModel
    {
        public string Id { get; set; }
        public string Content { get; set; }
        public string? AuthorId { get; set; }
        public string PostId { get; set; }
        public string CreationDate { get; set; }
    }
}
